### ISSUE TITLE
Some feedback

### ISSUE THREAD

**Issue by jscaetano** [lostella/podds#1]
Nice code but it got some errors.

If i have a hand of say **As Ac**, _pre-flop_, with say, 6 players, the odds of having a **pair** show up as **0.357**, shouldnt it be **1.000**? given the fact that i already have a pair in hand? and the odds of having **two pairs** is **0.359**, so it is more probable of having two pairs than one, if already have one? that is not right.

Also, in the same example, hand of **As Ac**, _post-flop_, being the flop **Ah Ks Tc** for example, i have a **three of a kind**, but the odds of having **three of a kind** show up as **0.612**, once again, shouldnt it be **1.000**?

But if i have a hand of **As Ac**, and the flop is **Ah Ad Ks**, the four of a kind odd is **1.000**, in that case is right.

That was the only mistake i found on this code. if i find more I'll submit another issue. Looking forward for the fix :) thanks :)

**Comment by lostella** [comment]
> If i have a hand of say As Ac, pre-flop, with say, 6 players, the odds of having a pair show up as 0.357, shouldnt it be 1.000?

I don’t think so, with a pair in hand you might as well end up with a three- or four-of-kind, as well as other things.

(I’m not saying 0.357 is necessarily correct, but 1.000 is not)

**Comment by jscaetano** [comment]
Hey there, I have a question. Could your code be adapted to incline ranges?
like if i have an idea of the range of the opponent??

Lorenzo Stella ***@***.***> escreveu no dia terça, 21/06/2022
à(s) 06:01:

**Comment by jscaetano** [comment]
Ohh!! Okay I understand now! Thanks for the fast feedback!

That makes sense! Its the probability of a pair being the best thing you
have.

Great code once again!

Lorenzo Stella ***@***.***> escreveu em ter., 21/06/2022 às
06:01 :
